### PR TITLE
Fix a crash in backup driver

### PR DESCRIPTION
**Pull request by xing-yang**
Signed-off-by: xing-yang <xingyang105@gmail.com>

This PR fixes a crash in the backupdriver controller.

### CODE REVIEW

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/5/

File: cmd/backupdriver/main.go [hunk 1]
```diff
@@ -115,9 +128,5 @@ func newK8sClient(master, kubeConfig string) (kubernetes.Interface, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to create config: %v", err)
```
Comment (dsu-igeek) [inline cmd/backupdriver/main.go:129]: Can we use errors.Wrapf() here?  Or we can start to use fmt.Errorf with %w and start a task to switch the rest of the code base over.

Comment (xing-yang) [reply cmd/backupdriver/main.go:129]: Sure.

Comment (xing-yang) [reply cmd/backupdriver/main.go]: Fixed.

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/6/

**Comment by dsu-igeek** [comment]
Test failures look like they are test problems, this fix should have no affect on the existing driver.

**Pull request merged**